source/scpd/tests/GlueTypes.d

```diff
@@ -59,6 +65,20 @@ alias TypesNoLayout = AliasSeq!
     xvector!Value,
     xvector!PublicKey,
     xvector!SCPQuorumSet,
+
+    // todo: these still don't match perfectly. Need to fix
+    // some other C++ bindings for this to work. Most of these
+    // types are not passed by value so it's currently ok.
+    //ValueWrapperPtr,
+    //BallotProtocol.SCPBallotWrapper,
+    //SCPEnvelopeWrapper,
+    //ValueWrapper,
+
+    //LocalNode,
+    //BallotProtocol,
+    //NominationProtocol,
+    //SCP,
+    //Slot,
```

Comment:
Size is off ?

Reply:
I've re-enabled most of them except two, which I haven't been able to match the size of on mac/linux reliably without guessing the size. Maybe @omerfirmak wants to take a look at these later.